keras_cv/utils/preprocessing.py

```diff
@@ -16,6 +16,37 @@
 
 from keras_cv import core
 
+_TF_INTERPOLATION_METHODS = {
+    "bilinear": tf.image.ResizeMethod.BILINEAR,
+    "nearest": tf.image.ResizeMethod.NEAREST_NEIGHBOR,
+    "bicubic": tf.image.ResizeMethod.BICUBIC,
+    "area": tf.image.ResizeMethod.AREA,
+    "lanczos3": tf.image.ResizeMethod.LANCZOS3,
+    "lanczos5": tf.image.ResizeMethod.LANCZOS5,
+    "gaussian": tf.image.ResizeMethod.GAUSSIAN,
+    "mitchellcubic": tf.image.ResizeMethod.MITCHELLCUBIC,
+}
+
+
+def get_interpolation(interpolation):
```

Comment:
just remove this file?

Reply:
All of preprocessing.py?  It has a lot of utilities in it right now.

Reply:
Actually, I'll need the same logic in CropAndResize - lets just keep it.  I think it is actually somewhat useful for consistency across all layers.